chain/network/src/peer_manager.rs

```diff
@@ -281,7 +281,7 @@ impl Actor for PeerManagerActor {
         if let Some(server_addr) = self.config.addr {
             // TODO: for now crashes if server didn't start.
             let listener = TcpListener::bind(&server_addr).unwrap();
-            info!(target: "network", "Server listening at {}@{}", self.peer_id, server_addr);
+            info!(target: "info", "Server listening at {}@{}", self.peer_id, server_addr);
```

Comment:
Why is target `info`?

Reply:
To show it at non verbose mode.

Reply:
I see. That feels weird. I think we shouldn't filter out `info` for any target